services/tests/twitter.test.js

```diff
@@ -0,0 +1,11 @@
+import TwitterService from "../twitter";
```

Comment:
rather than having a `tests` folder, we want to have the tests live side-by-side with the component/service the tests are for. 
This makes it easier find tests associated with the component you're working on and also to see which files have associated tests.

Reply:
Not sure I agree with that as the codebase can scale and making a single folder getting very big. For example, the components folder (that's already big).

But I think both can work, it's just a matter of convention and gets used to it. Update the test to be inline :)

Reply:
yes, we've planned to break components up into subfolders once it was large enough. (which now is that time 😃 ) 
When we had less than 10 components, it didn't make sense to add a layer of abstraction (folders) yet.